packages/backup-restore-operator/package.yaml

```diff
@@ -0,0 +1,5 @@
+url: 
```

Comment:
Yeah you need to fix CI so that this doesn't error out.

Reply:
yup, I don't think it's decided yet what the expected value should be?

Reply:
yes, I don't think its decided yet what the value for `url` should be for the scripts in case of original chart

Reply:
yes, it's being tracked in this issue https://github.com/rancher/rancher/issues/28628